spec/virtual_total_spec.rb

```diff
@@ -88,12 +88,12 @@
 
     context "with order clauses in the relation" do
       it "sorts by total" do
-        skip("fix order in scopes") if ENV["DB"] == "pg"
```

Comment:
I guess my only concern with this change is with these specs.

I don't have the full context, but were these working on other DB's, but then now aren't with PG?  Are we only testing against SQLite and PG?  If so, then was SQLite the one that was allowing the `ORDER BY`?  But then why did we change the values below in the `.create_books` calls?

Reply:
good questions.

1. in miq, we were only testing sqlite
2. in the gem, we started testing mysql and sqlite3 for this test (pg for others too)
3. with this change, we are not testing for all 3

create_books changed because in pg, with sorting, nulls are last, but in sqlite and mysql null is first. The change over there removed the null value so we could have an actual number there, and we'd avoid the nuance with null last (I was trying to focus on not blowing up. the nuance of a null was beyond the scope of the test)

Reply:
> create_books changed because in pg, with sorting, nulls are last, but in sqlite and mysql null is first.

So, the first arg in `create_books` (since [I wrote it](https://github.com/NickLaMuro/manageiq/commit/f8dddabc)) is just changing the number of books being created.  So I guess with how this was done previously (since every author seemed to have at least 1 published book), I don't see why it is necessary to up the counts.

Am I missing something?

Reply:
ok. removed those extra changes, just added the `author1.create_books` to avoid the null

Reply:
I had upped the count so there were no authors with no published books.
it made the sorting more consistent.

since then, I've put in a pr so it always returns a 0, so this is not an issue